student-store-ui/src/components/OrderCard/OrderCard.css

```
@@ -0,0 +1,65 @@
+.order-card {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    background: #fff;
+    border: 1px solid #e4e4e4;
+    padding: 20px 20px;
```

Nitpick on the edge of pedantry, but I think it's nice to order CSS alphabetically just so it's easier to skim and find a certain field.
```suggestion
    align-items: flex-start;
    background: #fff;
    border: 1px solid #e4e4e4;
    display: flex;
    flex-direction: column;
    padding: 20px;
```